test/integrations/huggingfaceDatasets.test.ts

```diff
@@ -9,20 +9,20 @@ jest.mock('../../src/fetch', () => ({
 }));
 
 describe('huggingfaceDatasets', () => {
-  let originalHfApiToken: string | undefined;
+  // Save original environment variables
+  const originalEnv = { ...process.env };
```

Comment:
How is this change related?

Reply:
This test started failing when I changed how the env variables were read